### ISSUE TITLE
Question about Weight

### ISSUE THREAD

**Issue by soon-will** [imatge-upc/retrieval-2017-cam#7]
Hi, thanks for your effort.
In the paper, 

> Optionally, an additional convolutional layer can be added before the GAP (CAM layer) to recover the accuracy drop after removing the fully-connected layers.

It means that there is a new conv layer is added, so how do you train the new network?
Do you only train the new conv layer? Which dataset do you use to train the network?ImageNet, or Oxford buildings?

Thanks for your reply.

**Comment by jsalbert** [comment]
Hey @soon-will ,

The network is trained on ImageNet dataset, not on the retrieval ones. 
It is trained from scratch, using the original trainings hyperparameters (of the respective networks).
